docs/Migration-Guide.md

```diff
@@ -140,4 +141,58 @@ You have to change the payment handling URL:
      The rest of the payment integration is similar to other redirect based payments, like PayPal or Sofortüberweisung.
      
      See more in [Field Mapping Guide](/docs/Field-Mapping.md)
-     
\ No newline at end of file
+
+## To use `bancontact` and `iDeal` payment methods
+  
+  _This section could be skipped if this payment method is not used_
+  
+  1. Contact Payone Merchant support and activate `bancontact` and `iDeal` payment method in your account, 
+  supply them required values from Paydirekt Merchant account.
```

Comment:
Can you explain how Paydirekt comes into this context? I thought Paydirekt is also kind of a bank_transfer type. _(just for my knowledge)_

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/313/commits/aebf8518c0557cdb46eab50ede11a52dfeed8c3e